### ISSUE TITLE
bad version in package.json

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/molarity#65]
On 2/27/19 in https://github.com/phetsims/molarity/commit/008761f645d25a20e636ef43433d2076179731d3, package.json in master was changed to:

```js
  "version": "1.5.0-screenSummary.1",
```

... for a on-off version.  This form of version identifier does not belong in master.  On-off version ids need to be removed immediately after publishing.  The correct dev version id should be restored.

**Comment by pixelzoom** [comment]
@twant please review. Mentioning @zepumph because I seem to recall him being associated with this on-off.

**Comment by zepumph** [comment]
@twant and I have not yet gone over building/package work. We will talk this through today! As far as the problem goes, it was totally my bad.
